public/common/utils.py

```diff
@@ -3142,18 +3146,21 @@ def get_tiles(
     if target_num_tiles is not None and target_num_tiles < 1:
         raise ValueError("target_num_tiles should be more than zero.")
 
-    if target_num_tiles == 1:
-        
+    if target_num_tiles == 1 and not equator_crossing:
         tile = mercantile.bounding_tile(*bounds.total_bounds)
         if verbose: 
             print(to_gdf((0,0,0)))
         gdf = to_gdf((tile.x, tile.y, tile.z))
     else:
-        zoom_level = (
-            zoom
-            if zoom is not None
-            else estimate_zoom(bounds, target_num_tiles=target_num_tiles)
-        )
+        if equator_crossing and target_num_tiles == 1:
```

Comment:
Nit: match structure above for readability
```suggestion
        if arget_num_tiles == 1 and equator_crossing:
```